mkdocs.yml

```diff
@@ -14,6 +14,7 @@ theme:
     repo: fontawesome/brands/github
     admonition:
       danger: octicons/alert-fill-12
+    annotation: material/eye-circle
```

Comment:
Ca fait quoi ça ?

Reply:
C'était pour les [annotations](https://squidfunk.github.io/mkdocs-material/reference/annotations/) pareil pour le `md_in_html` dans `markdown_extensions`. Cependant je n'ai pas réussi à le faire fonctionner donc je suis passée par les balises "spoiler".  J'ai completement zappé de les retirer.

Reply:
j'ai retiré

Reply:
> Ah ! j'avais cherché mais pas trouvé le lien que tu mets là ! Ca a l'air bien quand même ! je vais regarder, il y a possiblement un problème de compatibilité de versions

J'ai essayé de tester avec un mkdocs avec uniquement les dépendances dont j'avais besoin mais je n'ai pas réussi et il me semble qu'on est bon au niveau des versions. Après c'est noté comme "expérimental" mais comme y'a pas mal de monde qui l'utilisent (vu le nombre d'issue github sur le sujet) je pense quand meme que cela doit etre possible